src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/conv_sum_broadcast.cpp

```diff
@@ -388,6 +405,12 @@ const std::vector<fusingSpecificParams> fusingParamsSetBF16{
         fusingReluScaleShift
 };
 
+const std::vector<fusingSpecificParams> fusingParamsSetFP16{
+        emptyFusingSpec,
+        fusingSigmoid,
+        fusingReluScaleShift
+};
+
```

Comment:
What is the purpose of copying  this vector, it's identical to the existing `fusingParamsSetBF16`

Reply:
already updated it